packages/tasks/src/snippets/curl.ts

```diff
@@ -27,6 +27,32 @@ export const snippetTextGeneration = (model: ModelDataMinimal, accessToken: stri
 	}
 };
 
+export const snippetImageTextToTextGeneration = (model: ModelDataMinimal, accessToken: string): string => {
+	if (model.config?.tokenizer_config?.chat_template) {
```

Comment:
use model.tags.include("conversational") to be consistent with what @mishig25 did? (i think)

Reply:
`model.tags.include("conversational")`

yes, thats what I did

Reply:
I took the same as what existed for `textGeneration`. If I try to access "tags", I'm getting

```
Property 'tags' does not exist on type 'ModelDataMinimal'.
```

See https://github.com/huggingface/huggingface.js/blob/fae585340d11fb4f7f034387e19e275134e8d882/packages/tasks/src/snippets/types.ts#L8

Reply:
sure

Reply:
right, correct. Addressed in https://github.com/huggingface/huggingface.js/pull/927/commits/9cc3d868ec3fca027ec94cce3026f37bfc8c14ba